VogtPayroll2/Employee1.cs

```diff
@@ -0,0 +1,93 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace VogtPayroll2
+{
+    abstract class Employee
+    {
+        //For simplicity, use a constant tax rate of 30% to compute the tax amount
+        protected string name;
+        protected int hoursWorked;
+        protected decimal payRate;
+        protected const decimal taxAmount = .30m;
+
+        public Employee(string aName, int aHoursWorked, decimal aPayRate)
+        {
+            name = aName;
```

Comment:
Make the change from my last comment, then you can get rid of the `a` before each parameter. I feel like I've seen developers complain about prefixes to function parameters/class variables and the one I've seen most often... database field names.

By renaming your class variable you shouldnt have a conflict.

Name = name;
HoursWorked = hoursWorked;